doc/api/fs.md

```diff
@@ -378,7 +378,8 @@ fs.appendFile('message.txt', 'data to append', 'utf8', callback);
 
 Any specified file descriptor has to have been opened for appending.
 
-_Note: Specified file descriptors will not be closed automatically._
+_Note: If a file descriptor is specified as the `file`, it will not be closed 
+automatically._
```

Comment:
It was in the original text. It makes perfect sense. It's just one word and doesn't overload text or can be a source of confusion. Why does it have to be removed?..